jupyterlab_caip_optimizer/src/components/study_details.tsx

```diff
@@ -0,0 +1,297 @@
+  studyId,
+  studyToDisplay,
+  openTrials,
+  openVisualizations,
+  openDashboard,
+}) => {
+  const classes = useStyles();
+  const configRows = createConfigRows(studyToDisplay);
+  const paramRows = createParamRows(studyToDisplay);
+  return (
+    <Box m={3}>
+      <React.Fragment>
+        <Typography variant="h5" gutterBottom align="left">
+          {studyToDisplay.name}
+        </Typography>
+        <Grid container item spacing={1} xs={12}>
+          <Grid container item sm={9}>
+            <TableContainer component={Paper}>
+              <Table
+                className={classes.table}
+                aria-label="customized table"
+                size="small"
+              >
+                <TableHead>
+                  <TableRow>
+                    <StyledTableCell>Study Configuration</StyledTableCell>
+                    <StyledTableCell align="right" />
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  {configRows.map((row: ConfigRow, index: number) => (
+                    <StyledTableRow key={`config-row-${index}`}>
+                      <StyledTableCell
+                        key={`config-row-${index}-cell-entryName`}
+                        component="th"
+                        scope="row"
+                      >
+                        {row.entryName}
+                      </StyledTableCell>
+                      <StyledTableCell
+                        key={`config-row-${index}-cell-entryValue`}
+                        align="right"
+                      >
+                        {row.entryValue}
+                      </StyledTableCell>
+                    </StyledTableRow>
+                  ))}
+                </TableBody>
+              </Table>
+            </TableContainer>
+            <TableContainer component={Paper}>
+              <Table
+                className={classes.table}
+                aria-label="customized table"
+                size="small"
+              >
+                <TableHead>
+                  <TableRow>
+                    <StyledTableCell>Parameters</StyledTableCell>
+                    <StyledTableCell align="right">Type</StyledTableCell>
+                    <StyledTableCell align="right">Values</StyledTableCell>
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  {paramRows.map((row: ParamRow, index: number) => {
+                    return (
+                      <StyledTableRow key={`param-row-${index}`}>
+                        <StyledTableCell
+                          key={`param-row-${index}-cell-name`}
+                          component="th"
+                          scope="row"
+                        >
+                          {row.name}
+                        </StyledTableCell>
+                        <StyledTableCell
+                          key={`param-row-${index}-cell-type`}
```

Comment:
You only need a key for the top most element in a map function. These sub `key` values are not needed.

Reply:
fixed!